membership-attribute-service/app/services/AttributesFromZuora.scala

```diff
@@ -52,13 +55,24 @@ class AttributesFromZuora(implicit val executionContext: ExecutionContext, syste
           case Failure(e) => SafeLogger.error(scrub"Failed to retrieve attributes from cache for $identityId because $e")
         }
 
+    def userHasDigiSub(accountsWithSubscriptions: List[AccountWithSubscriptions]) =
+      accountsWithSubscriptions.flatMap(_.subscriptions).exists(_.asDigipack.isDefined)
```

Comment:
it's sensible to take this small optimisation. There is an even smaller additional optimisation, if they have a Paper+Digital subscription.